lego/apps/users/fixtures/test_abakus_groups.yaml

```diff
@@ -92,11 +92,13 @@
 - model: users.abakusgroup
   pk: 19
   fields: {deleted: false, name: Bedkom, description: '', parent: 18, logo: null,
-    type: komite, text: '', permissions: '[]', lft: 3, rght: 4, tree_id: 15, level: 2}
+    type: komite, text: '', permissions: '["/sudo/admin/companyinterest/", "/sudo/admin/surveys/",
+    "/sudo/admin/submissions/"]', lft: 3, rght: 4, tree_id: 15, level: 2}
```

Comment:
These will have to be added in prod manually, right?

Reply:
Yep

Reply:
Remind me!